tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+requires =
+    tox>=4
+env_list = py{38,39,310,311,312}
+
+[testenv]
+deps = pytest
+commands = pytest
+extras = dev
+set_env =
+  VIRTUALENV_DISCOVERY = pyenv
```

Comment:
It doesn't appear required, at least on my windows environment. I had a lot of trouble getting tox to run initially, and it may have been a leftover of experimentation.